packages/formation/sass/site/_m-vet-nav.scss

```diff
@@ -207,7 +207,7 @@ body.va-pos-fixed {
     }
   }
 
-  &[aria-expanded='true'] {
+  &[aria-expanded="true"] {
```

Comment:
guess we need to add scss linting to this repo.

Reply:
Spoke to @jbalboni and moved the changes to this file over to the vet-website PR, https://github.com/department-of-veterans-affairs/vets-website/pull/9799/files.